tests/jsonschema/bottle.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://formulae.brew.sh/api/bottle",
```

Comment:
Can remove this, it's going away in https://github.com/Homebrew/formulae.brew.sh/pull/702

Reply:
Always great to know you put in work for nothing